frontend/packages/client/src/components/WalletConnectModal.js

```diff
@@ -40,7 +33,7 @@ export default function WalletConnectModal({
             );
             closeModal();
           },
-          icon: getWalletIcon(service.provider),
```

Comment:
maybe doing this:
```
 icon: IS_LOCAL_DEV
            ? getWalletIcon(service.provider)
            : service.provider.icon,

Reply:
Ok updated @germanurrus . Just out of curiosity, why is it like that on local dev instead of just using the Discovery API with testnet services?

Reply:
We are developing in local dev using emulator for connecting wallets, this way we can deploy contracts to a controlled env that gives us more flexibility to develop